src/setup/node.rs

```diff
@@ -67,11 +67,35 @@ impl Node {
         let config = NodeConfig::new();
         let meta = NodeMetaData::new()?;
 
-        Ok(Self {
+        let mut node = Self {
             config,
             meta,
             process: None,
-        })
+        };
+
+        // insert the config file cmd args
+        match node.meta.kind {
+            NodeKind::Zebra => {
+                // Bit more convoluted since we need to insert the args before `start`, which comes last.
+                let start_arg = node
+                    .meta
+                    .start_args
+                    .pop()
+                    .expect("Expected at least one arg for Zebra (`start`)");
+                node.meta.start_args.push("--config".into());
+                node.meta
+                    .start_args
+                    .push(node.config_filepath().into_os_string());
```

Comment:
These two previous steps could probably be condensed (perhaps with `format!` and `OsString::from`)?

Reply:
For some reason combining the two into one, results in extra quotes around just that pair of args which breaks the command.

So instead of
```
`arg1 arg2 arg3 arg4 start`
```
we get

```
`arg1 arg2 `arg3 arg4` start`
```